src/GerryChain.jl

```diff
@@ -22,7 +23,7 @@ get_subgraph_population,
 induced_subgraph_edges, update_partition_adjacency,
 
 # balance edges
-random_kruskal_mst, 
+random_kruskal_mst,
```

Comment:
IDK why there is a diff here? I think perhaps because I deleted a superfluous space at the end of the line?